she_codes_news/news/models.py

```diff
@@ -1,8 +1,26 @@
+from django.contrib.auth import get_user_model
 from django.db import models
 
+USER = get_user_model()
+
 
 class NewsStory(models.Model):
     title = models.CharField(max_length=200)
-    author = models.CharField(max_length=200)
+   # author = models.CharField(max_length=200)
+    author = models.ForeignKey(
+        USER, on_delete=models.CASCADE, related_name="stories"
+    )
     pub_date = models.DateTimeField()
     content = models.TextField()
+    class Meta:
+        ordering = ["-pub_date"]
```

Comment:
i tend to avoid putting filtering on the model, i think it is better placed at the query level